packages/ember-debug/lib/index.js

```diff
@@ -130,7 +136,9 @@ if (DEBUG) {
     @private
   */
   setDebugFunction('info', function info() {
-    Logger.info.apply(undefined, arguments);
+    /* eslint-disable no-console */
```

Comment:
@For some of these disablings you may be able to make it a tad less noisy via `// eslint-disable-next-line no-console`